Scripts/run_test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+source ${HOME}/.bashrc
```

Comment:
Why is this needed?

Reply:
Technically this will be run whenever a terminal is started up. I believe it was added for testing whether the environment variables were actually set. I will remove it.